hcaptcha/src/main/java/com/stripe/hcaptcha/HCaptcha.kt

```diff
@@ -127,13 +126,10 @@ class HCaptcha private constructor(
          * @param activity The current activity
          * @return new [HCaptcha] object
          */
-        fun getClient(activity: FragmentActivity): HCaptcha {
-            return HCaptcha(activity, HCaptchaInternalConfig())
-        }
-
+        @JvmOverloads
```

Comment:
Do we need this?

Reply:
probably not since this is never referenced from java. Can remove.